modules/api/src/main/java/org/apache/ignite/table/TupleImpl.java

```diff
@@ -180,6 +180,18 @@ public BinaryObject binaryObjectValue(int columnIndex) {
         return value(columnIndex);
     }
 
+    /** {@inheritDoc} */
+    @Override
+    public boolean booleanValue(@NotNull String columnName) {
```

Comment:
I suggest to remove NotNull annotation for whole file sue to our code style rules prohibit to use it - https://cwiki.apache.org/confluence/display/IGNITE/Java+Code+Style+Guide#JavaCodeStyleGuide-2.1Commonannotatingrules